scripts/mkcloud

```diff
@@ -540,24 +540,84 @@ function setupadmin()
     fi
 }
 
+function shutdown_node() {
+    virsh shutdown $1
+}
+
+function destroy_node() {
+    virsh destroy $1
+}
```

Comment:
Same argument here. This isolates the virtualization details into these two functions.